src/clientdlg.cpp

```diff
@@ -1354,3 +1357,53 @@ rbtReverbSelR->setStyleSheet ( "" );
     // also apply GUI design to child GUI controls
     MainMixerBoard->SetGUIDesign ( eNewDesign );
 }
+
+void CClientDlg::OnRecorderStateReceived (  const ERecorderState newRecorderState )
+{
+    MainMixerBoard->SetRecorderState ( newRecorderState );
+    SetMixerBoardDeco ( newRecorderState, pClient->GetGUIDesign() );
+}
+
+void CClientDlg::OnGUIDesignChanged()
+{
+    SetGUIDesign ( pClient->GetGUIDesign() );
+    SetMixerBoardDeco ( MainMixerBoard->GetRecorderState(), pClient->GetGUIDesign() );
+}
+
+void CClientDlg::SetMixerBoardDeco(  const ERecorderState newRecorderState, const EGUIDesign eNewDesign  )
+{
+    // return if no change
+    if( (newRecorderState == eLastRecorderState) && (eNewDesign == eLastDesign) )
```

Comment:
I think you'll need a space between if and ( and also between ( and newRecorderState etc.